src/main/java/xyz/nucleoid/plasmid/game/stats/StatisticKey.java

```diff
@@ -43,6 +43,14 @@ public JsonObject encodeValue(T value) {
         return obj;
     }
 
+    public String getTranslationKey() {
```

Comment:
Might it be a bit confusing for users of this API to know which translation key method to use?

Reply:
Hmm, I could add a javadoc explaining the purpose of the two methods. Ideally it would be one method, but I'm not sure if there's a clean way to test if a translation key exists or not, so at the minute its left up to the caller

Reply:
Pushed some javadoc that should explain the difference